sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -94,6 +95,15 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
     },
 
     render: function() {
-        this.$el.html(this.template({ item: this.model }));
+        var shownCategories;
+        if (this.user.get('email')) {
+            shownCategories = this.model.categories.filter(function(category) {
+              return category.get("item_count") > 1;
+            });
+        } else {
+            shownCategories = this.model.categories.first(Sourcemash.Views.ItemCardView.MAX_CATEGORIES_SHOWN)
+        }
+
+        this.$el.html(this.template({ item: this.model, categories: shownCategories }));
     }
-});
+}, {MAX_CATEGORIES_SHOWN: 5});
```

Comment:
Right. The second dictionary just allows you to pass whatever values you want to be added to the view class. It would be the same as if I typed ```Sourcemash.Views.ItemCardView.MAX_CATEGORIES = 5```. It adds the property onto all instances of the view so that I can access it like above.